poetry.lock

```diff
@@ -1,26 +1,24 @@
-# This file is automatically @generated by Poetry 1.4.2 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
```

Comment:
Granted, `1.6.1` is better than `1.4.2`, but I think we've been using a `1.8.x` version, normally.  (I think we've been resisting going to `2.x`....)